published/064-2019-oct-22.md

```diff
@@ -0,0 +1,69 @@
++ [Senior Golang Engineer](https://www.golangprojects.com/golang-go-job-def-Senior-Golang-Engineer-Remote-Textile.html), Textile, Remote
++ [Distributed Systems Technical Lead, IPFS](https://jobs.lever.co/protocol/9283f9b0-de64-4e1f-a221-5d02b0202198), Protocol Labs, Remote
+
+
+## Updates and new releases
+*See the latest releases of IPFS tools and projects across the ecosystem.*
+
++ [Qri Desktop 0.1.0 is OUT!](https://qri.io/desktop/) Version your datasets locally and publish them (with full commit history) to the cloud.
++ [ARK Core v2.6](https://blog.ark.io/ark-core-v2-6-is-now-live-on-devnet-bfd8798fadf3) is Now Live On Devnet
+
+## Tools and projects we <3
+*[Awesome IPFS](https://awesome.ipfs.io/) is a community maintained and updated list of projects, tools, or pretty much any things related to IPFS that are totally awesome. To see more, or add yours to the list, visit [Awesome IPFS on GitHub](https://github.com/ipfs/awesome-ipfs).* 
+
++ [Announcing lotus](https://filecoin.io/blog/announcing-lotus/), the first alternate Filecoin implementation!
++ [3Box Ghost Threads API](https://docs.3box.io/build/web-apps/messaging/ghost-threads): Ghost threads are a different type of thread that doesn't persist any messages to database storage. Instead, messages are sent from one peer to the rest of the network using IPFS pubsub, and upon receipt by other online peers are kept in-memory.
++ [Large](https://gitlab.com/ptoner/large) is a P2P blogging platform. Build an online community that's run and hosted by the community.
++ [Permaweb](https://github.com/Permaweb/permaweb) runs on a series of self-hosted nodes. These are powered by Textile.io, which is acts as a digital wallet for fully encrypted, decentralized content. Content is stored using the IPFS protocol, which makes Permaweb links unique and permanent. 
+
+
+## Coming up in the Community
+*Did you know IPFS has a community forum at [discuss.ipfs.io](https://discuss.ipfs.io/)? Sign up to participate in discussions about coding, tutorials, see announcements and learn about upcoming community events.*
+
++ **25 October 2019:** [Re­decentra­lize Conference 2019](https://redecentralize.org/events/2019-conference/) Interested in a fairer, freer web and net? Working on decentralisation? Care about privacy, automomy and our future digital infrastructure? Then join us to learn, compare experiences, bounce ideas, share challenges and lessons learnt, hatch collaborations, plan campaigns and start projects. London, UK.
++ **27 October 2019:** [Force Commercial Platform Based on IPFS](https://www.meetup.com/Shanghai-Decentralized-Systems-Meetup-Group/events/265612157/) hosted by IPFS Force, will focus on studying Decentralized Storage Network built on IPFS by Protocol Labs and developing services and application based tools in IPFS bundle. Shanghai, China.
```

Comment:
Suggested rewording to make it more clear that the talk is about IPFS Force, not about IPFS itself: 

`**27 October 2019:** [Force Commercial Platform Based on IPFS](https://www.meetup.com/Shanghai-Decentralized-Systems-Meetup-Group/events/265612157/) This meetup hosted by the IPFS Force Community will introduce the IPFS Force commercial platform, which connects decentralized storage providers and users on top of the IPFS network. Shanghai, China.`